DecipherNow/gm-fabric-dashboard
# WIP: 108 filter out metrics that have values of only zero

Resolves #108 

## Proposed Changes

  - add two checkboxes to filter metrics in Explorer view 
 - `Hide all metrics with only zero values`
 - `Hide all static metrics`
 - checking `Hide all static metrics` automatically check `Hide all metrics with only zero values` and disable it
 - add more unit tests to Explorer 
 - add styled components test
 - checkbox filters are router driven and retained along with search query on hard refresh
  - styling is TBD
## How to Test

  - Choose any Go/JVM Instance and navigate to Explorer View.  
  - check and uncheck both checkboxes to confirm metrics are filtered appropriately
  - run npm t
